src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -66,7 +66,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('collectors.xml');
         $loader->load('guard.xml');
 
-        if ($container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
+        if ($container->getParameter('kernel.debug')) {
             $loader->load('security_debug.xml');
```

Comment:
@stof you're right, then

https://github.com/symfony/symfony/pull/18934#discussion_r65501397
> ... Only the parameters are copied from actual to tmp container ...

We can create a new parameter something like `profiler.collect` into [`FrameworkExtension::registerProfilerConfiguration`](https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L291) ?:
```php
$container->setParameter('profiler.collect', $config['collect']);
```
and use it here ?
```php
if ($container->hasParameter('profiler.collect') && $container->getParameter('profiler.collect')) {
    $loader->load('collectors.xml');
    $loader->load('security_debug.xml');
}
```
@javiereguiluz WDYT?